WordPress/Classes/ViewRelated/Blog/SettingTableViewCell.h

```diff
@@ -0,0 +1,5 @@
+#import "WPTableViewCell.h"
+
+@interface SettingTableViewCell : WPTableViewCell
```

Comment:
Minor thing, but I think this should be named `SettingsTableViewCell` for consistency. Although if you look at our existing custom cells half of them use the singular and the other half use plural :disappointed: